packages/vuetify/src/components/VAutocomplete/VAutocomplete.ts

```diff
@@ -168,7 +168,8 @@ export default VSelect.extend({
         document.addEventListener('copy', this.onCopy)
         this.$refs.input && this.$refs.input.select()
       } else {
-        document.removeEventListener('copy', this.onCopy)
+        document.removeEventListener('copy', this.onCopy);
+        (this.$_menuProps as any).closeOnContentClick && this.$refs.input.blur()
```

Comment:
```suggestion
        document.removeEventListener('copy', this.onCopy)
        ;(this.$_menuProps as any).closeOnContentClick && this.$refs.input.blur()
```

Reply:
Does this have to be specific to `closeOnContentClick`? Is there any reason we wouldn't want to always make sure the input is blurred when we lose focus?

Reply:
@KaelWD yep You are right, we do not have a reason for that. I was focused on parent issue to much.